src/ui/components/Timeline/UnloadedRegions.tsx

```diff
@@ -31,7 +27,21 @@ export const UnloadedRegions: FC = () => {
   let beginTime = begin.time;
   let endTime = end.time;
   if (focusRegion) {
-    const overlappingRegions = overlap([rangeForFocusRegion(focusRegion)], loadedRegions.loading);
+    // Even though we technically focus on the nearest points,
+    // We should only show the user a range defining their specified times.
+    const unsafe = focusRegion as UnsafeFocusRegion;
+    const userVisibleTimeStampedPointRange: TimeStampedPointRange = {
+      begin: {
+        point: unsafe.begin.point,
+        time: unsafe.beginTime,
+      },
+      end: {
+        point: unsafe.end.point,
+        time: unsafe.endTime,
+      },
+    };
```

Comment:
This fixes an issue on the `main` branch where our mini-timeline showed a _larger_ loaded region than the user had focused on, which was confusing.

Before:
![Screen Shot 2022-09-13 at 12 08 21 PM](https://user-images.githubusercontent.com/29597/189952248-a0bf1571-a822-4ae8-a48d-0b7a0686f020.png)

After:
![Screen Shot 2022-09-13 at 12 08 42 PM](https://user-images.githubusercontent.com/29597/189952304-31d3bd11-17fb-4a86-837c-dcd9a406949b.png)